protoc-gen-go/grpc/grpc.go

```diff
@@ -256,6 +260,7 @@ func (g *grpc) generateService(file *generator.FileDescriptor, service *pb.Servi
 	g.P("},")
 	g.P("Metadata: \"", file.GetName(), "\",")
```

Comment:
I have a concern about this field.  This is supposed to be extensible, for gRPC internal use.  Exporting this struct means users may begin to rely upon this behavior.  Is there a way to prevent that?